colour/utilities/common.py

```diff
@@ -302,3 +305,48 @@ def is_integer(x):
     """
 
     return abs(x - round(x)) <= INTEGER_THRESHOLD
+
+
+def filter_kwargs(function, **kwargs):
```

Comment:
You mean so you can pass the same set of `kwargs` to both of them and not get an error?